cmd/metrics-node-sampler/cmd/cmd.go

```diff
@@ -109,7 +120,7 @@ func init() {
 	_ = RootCmd.MarkFlagRequired("sampler-config-filepath")
 	RootCmd.Flags().StringVar(&logPath, "log-level-filepath", "", "path to log level file.  The file must contain a single integer corresponding to the log level (e.g. 2")
 	RootCmd.Flags().IntVar(&terminationSeconds, "termination-seconds", 10, "time to wait for shutdown before os.Exit is called")
-
+	RootCmd.Flags().StringVar(&S.hostName, "host-name", "", "DNS resolvable kubelet host name if different from the NODE_NAME")
```

Comment:
It seems like this is supposed to be a path to a file containing the hostname, not the hostname itself? I would rename the flag to make that clear